### DISCUSSION TITLE
I am excited about the prospect of joining your team and bringing my proven track record of delivering high-quality solutions.

### DISCUSSION THREAD

**Discussion by 0x0factor** [Sparganothis/Sparganothis.github.io#2] [General]
Hi, I'm Ethan Tan, a web full-stack and blockchain developer with extensive experience spanning over 8 years. I specialize in developing resilient, scalable web applications and state-of-the-art blockchain solutions. My expertise in frameworks like React.js, Vue.js, and Next.js, along with my proficiency in Solidity for smart contract development, ensures I deliver premium, secure, and efficient solutions tailored to your specific needs.

**Comment by GithubCosmin** [comment]
Hi 

This project is pure rust - we don't want to add crypto-coins but we are insterested in anti-cheat p2p methods - maybe we can host some kind of chain to verify the game traffic

Here are some resources to get started with the game

https://book.leptos.dev/

**Reply by 0x0factor** [reply]
good

-Contact Info
skype: https://join.skype.com/invite/FVUI6gLcYgAE
telegram: https://t.me/ethantan1116
slack: https://join.slack.com/t/ksc-xme4972/shared_invite/zt-2ld45idp2-G9HkxP_wX4ctwQshyPM7Ig
discord: https://discord.gg/VxnfdY4bS4
